### DISCUSSION TITLE
[Windows 11] No light with Prime Video

### DISCUSSION THREAD

**Discussion by Yamh631** [awawa-dev/HyperHDR#936] [Q&A] [answered]
Hello,

I have HyperHDR 20.0 on my PC, it works with Youtube and Media Player.

**But no light with Prime Video on browser** :
I try with different browser : Brave, Firefox, Edge, Chrome)
I use "Software Screen Capture" mode on HyperHDR with GTX1070 Ti

Is it because of copyright?
How to do it?

Thanks.

**Comment by awawa-dev** [comment, marked as answer]
Hi
Yes, it's copy protection and results in a black screen/rectangle in the HyperHDR preview. Same situation as Netflix. https://github.com/awawa-dev/HyperHDR/issues/688

**Reply by Yamh631** [reply]
Hi
Okay thanks

**Reply by dipanjanpanja6** [reply]
try this 

Go to Chrome → Settings → System.
Turn Use graphics acceleration when available OFF.
Restart Chrome.
